### ISSUE TITLE
Forgot Password Flow

### ISSUE THREAD

**Issue by Polsander** [ferncabrera/open_ims#63]
We have enough functions and architecture set-up to finish our forgot password flow. U.I. is already in place for the modal.
**Requirements**
- Create a POST api in server to handle email submission from user.
- Set-up and use an email service to email any found user instructions to reset their password (please talk to @kellygibson for text)
- Ensure that the reset password flow follows the figma.

No errors or any indication of an email not existing should reflect on the front-end. This makes it more secure.

[Figma link](https://www.figma.com/file/LzQ0oUTimkNupQm1U7z3dG/open-ims?type=design&node-id=118-87&mode=design&t=jpMnyOcdGoHJjt7n-0)

Some email services that could be good:
- Brevo (formerly sendInBlue, I've used it and it's really good).
**these may require some research if we would like to use these instead**
- Mailchimp
- Mailgun
- Postmark

I would avoid using SendGrid as that email service goes through a lot of updates and attacks that resulted in them locking my account for bot suspicion after 3 months of idle use. Their customer support is horrible as well and I will probably never get my account back since I have to actually make a phone call with them.
 TLDR: Sendgrid API might not work after some time and your account might be locked forever.

Let me know if you have any questions.

**Comment by Vayras** [comment]
hi I would like to contribute towards this issue

**Comment by VINAAAYYY** [comment]
Hi, I would like to contribute towards this issue. Please provide me access to the figma

**Comment by ferncabrera** [comment]
Sorry @Vayras and @VINAAAYYY we are not looking for outside help on tickets at the moment 😞 But feel free to fork and play around with what we have so far :)

**Comment by VINAAAYYY** [comment]
Alright, thanks for the heads up. Happy to help and learn if you need any help in the future.
